### PR TITLE
docs(rxOptionTable): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxOptionTable/scripts/rxOptionTable.page.js [hunk 1]
```diff
@@ -44,20 +53,26 @@ var optionRowFromElement = function (eleOptionRow) {
...
          * @function
+         * @instance
          * @memberof rxOptionTable.row
          * @description
          * Return the value of the cell by `columnName`, using `getText` by default.
          * For more control, pass in a `customFn`.
          * The reason `columnName` is used, as opposed to by binding, is due to some
          * complexity contained within the `getContent` function in the rxOptionTable directive.
-         * [Link to the `getContent` function]{@link http://goo.gl/HKBoez}.
+         * [Link to the `getContent` function]{@link https://goo.gl/jWMt2V}.
```
Comment (CITguy) [inline src/components/rxOptionTable/scripts/rxOptionTable.page.js]: :+1: for relinking to current source location

File: src/components/rxOptionTable/scripts/rxOptionTable.page.js [hunk 2]
```diff
@@ -395,24 +541,21 @@ var rxOptionTable = {
     }
 };
 
-/**
- * @deprecated
- * @namespace
- * @description
- * **DEPRECATED**: Please use {@link rxOptionTable} as a stand-in replacement.
- */
 var rxOptionFormTable = rxOptionTable; // jshint ignore:line
```
Comment (CITguy) [inline src/components/rxOptionTable/scripts/rxOptionTable.page.js]: Is jshint still complaining about something on this line?

**Comment by CITguy** [comment]
LGTM

File: src/components/rxOptionTable/scripts/rxOptionTable.page.js (hunk 2 above)
Comment (Droogans) [inline src/components/rxOptionTable/scripts/rxOptionTable.page.js]: I'll take a second look.

**Comment by jj4th** [comment]
QE-LGTM

**Pull request merged**